references.bib

```diff
@@ -0,0 +1,278 @@
+  year={2005}
+}
+%used
+@inproceedings{leigh2007using,
+  title={Using a genetic algorithm to explore A*-like pathfinding algorithms},
+  author={Leigh, Ryan and Louis, Sushil J and Miles, Chris},
+  booktitle={Computational Intelligence and Games, 2007. CIG 2007. IEEE Symposium on},
+  pages={72--79},
+  year={2007},
+  organization={IEEE}
+}
+%used
+@inproceedings{harabor2011online,
+  title={Online Graph Pruning for Pathfinding On Grid Maps.},
+  author={Harabor, Daniel Damir and Grastien, Alban and others},
+  booktitle={AAAI},
+  year={2011}
+}
+%used
+@book{millington2016artificial,
+  title={Artificial intelligence for games},
+  author={Millington, Ian and Funge, John},
+  year={2016},
+  publisher={CRC Press}
+}
+%used
+@article{wang2012game,
+  title={Game ai: Simulating car racing game by applying pathfinding algorithms},
+  author={Wang, Jung-Ying and Lin, Yong-Bin},
+  journal={International Journal of Machine Learning and Computing},
+  volume={2},
+  number={1},
+  pages={13},
+  year={2012},
+  publisher={IACSIT Press}
+}
+%used
+@misc{harabor2012fast,
+  title={Fast pathfinding via symmetry breaking},
+  author={Harabor, Daniel},
+  journal={2013-11-20]. http://aigamedev. com/open/tutorial/symmetry-in-pathfinding},
+  year={2012}
+}
+%used
+@inproceedings{bulitko2003lookahead,
+  title={Lookahead pathologies and meta-level control in real-time heuristic search},
+  author={Bulitko, Vadim},
+  booktitle={Proceedings of the 15th Euromicro Conference on Real-Time Systems},
+  pages={13--16},
+  year={2003}
+}
+%used
+@article{hart1968formal,
+  title={A formal basis for the heuristic determination of minimum cost paths},
+  author={Hart, Peter E and Nilsson, Nils J and Raphael, Bertram},
+  journal={IEEE transactions on Systems Science and Cybernetics},
+  volume={4},
+  number={2},
+  pages={100--107},
+  year={1968},
+  publisher={IEEE}
+}
+%used
+@inproceedings{ferguson2005guide,
+  title={A guide to heuristic-based path planning},
+  author={Ferguson, Dave and Likhachev, Maxim and Stentz, Anthony},
+  booktitle={Proceedings of the international workshop on planning under uncertainty for autonomous systems, international conference on automated planning and scheduling (ICAPS)},
+  pages={9--18},
+  year={2005}
+}
+%used
+@inproceedings{demyen2006efficient,
+  title={Efficient triangulation-based pathfinding},
+  author={Demyen, Douglas Jon and Buro, Michael},
+  booktitle={Aaai},
+  volume={6},
+  pages={942--947},
+  year={2006}
+}
+%used
+@inproceedings{hale2009full,
+  title={Full 3D Spatial Decomposition for the Generation of Navigation Meshes.},
+  author={Hale, D Hunter and Youngblood, G Michael},
+  booktitle={AIIDE},
+  year={2009}
+}
+%used
+@article{jansen2007hpa,
+  title={HPA* Enhancements.},
+  author={Jansen, M Renee and Buro, Michael},
+  journal={AIIDE},
+  volume={7},
+  pages={84--87},
+  year={2007}
+}
+%used
+@inproceedings{togelius20102009,
+  title={The 2009 mario ai competition},
```

Comment:
Incorrect capitalisation -- should be
```
title={The 2009 {Mario AI} Competition}
```
(the extra curly braces are to prevent BibTeX from messing with the capitalisation on those words)